mentat-vscode/src/utils/server.ts

```diff
@@ -77,28 +81,52 @@ class Server {
         );
         const pythonLocation = path.join(binFolder, "python");
 
-        // TODO: Auto update mentat if wrong version
         const { stdout } = await aexec(`${pythonLocation} -m pip show mentat`);
         const mentatVersion = stdout
             .split("\n")
             .at(1)
             ?.split("Version: ")
             ?.at(1);
-        if (mentatVersion === undefined) {
+        if (mentatVersion !== MENTAT_VERSION) {
             progress.report({ message: "Mentat: Installing..." });
-            await aexec(`${pythonLocation} -m pip install mentat`);
+            await aexec(
+                `${pythonLocation} -m pip install mentat==${MENTAT_VERSION}`
+            );
             console.log("Installed Mentat");
         }
 
         return binFolder;
     }
 
+    private collectConfigOptions(): string[] {
+        const configuration = vscode.workspace.getConfiguration("mentat");
+        const optionNames = [
```

Comment:
Agreed it would be a lot nicer. Seeing as we have to manually add the config options to the package.json right now anyways though I think we can hold off on this for a bit.